tools/myst/host/package.c

```diff
@@ -295,23 +295,24 @@ int _package(int argc, const char* argv[])
     if (signing_engine_path)
     {
         // optional path option for signing engine
-        const char* sign_engine_args[] = {argv[0],
-                                          "sign",
-                                          rootfs_file,
-                                          pem_file,
-                                          config_file,
-                                          "--pubkeys",
-                                          pubkeys_file,
-                                          "--roothashes",
-                                          roothashes_file,
-                                          "--outdir",
-                                          tmp_dir,
-                                          "--signing-engine-name",
-                                          signing_engine_name,
-                                          "--signing-engine-key",
-                                          signing_engine_key,
-                                          "--signing-engine-path",
-                                          signing_engine_path};
+        const char* sign_engine_args[] = {
+            argv[0],
+            "sign",
+            rootfs_file,
+            pem_file, // TODO: maybe I need to take this out
```

Comment:
Can you make this null and see if it works? You should not need a pem file.

Reply:
Does oesign need a pem file with the signing engine?

Reply:
It does not need one, null seems to work. Will remove the comment. The thing here is we use positional arguments, so if we do not have a pem file it would potentially be a breaking change for everyone..

Reply:
Can we let people specify "NULL" as a command line param.

Reply:
Definitely, they can already do that. Tested it out

Reply:
Added that to this test so we can maintain is as the recommended way to use signing engines